cocotb/_vendor/distutils_version.py

```diff
@@ -0,0 +1,339 @@
+#
+# distutils/version.py
+#
+# Implements multiple version numbering conventions for the
+# Python Module Distribution Utilities.
+#
+# $Id$
+#
```

Comment:
Does PSF require use to include the license with this file? That will also transitively reclassify the entire project as I think the PSF license is stricter than MIT. @imphil Are you more familiar with license issues?

Reply:
No license needs to be included, but please prepend our standard (cocotb) SPDX header and add a copyright line following https://github.com/python/cpython/blob/main/LICENSE#L86-L88.

Reply:
I think I got that right.